src/app/kvm/components/KVMHeaderForms/RefreshForm/RefreshForm.test.tsx

```diff
@@ -28,20 +25,16 @@ describe("RefreshForm", () => {
         }),
       }),
     });
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter initialEntries={[{ pathname: "/kvm", key: "testKey" }]}>
-          <CompatRouter>
-            <RefreshForm clearSidePanelContent={jest.fn()} hostIds={[1]} />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+
+    renderWithBrowserRouter(
+      <RefreshForm clearSidePanelContent={jest.fn()} hostIds={[1]} />,
+      { state, route: "/kvm" }
+    );
+
+    await userEvent.click(
+      screen.getByRole("button", { name: /Waiting for action to complete/i })
     );
```

Comment:
```suggestion
```

You can remove this `click`, since the state has already specified that the pod is refreshing. The test will still pass